mirage/config.js

```diff
@@ -589,6 +589,62 @@ export default function () {
   this.get('/v3/enterprise_license', function (schema, request) {
     return new Response(404, {}, {});
   });
+
+  this.get('/insights/metrics', function (schema, {queryParams}) {
+    queryParams.owner_id = parseInt(queryParams.owner_id);
+    queryParams.name = queryParams.name.split(',');
```

Comment:
I would suggest to avoid mutating function arguments.

```javascript
const ownerId = parseInt(queryParams.owner_id);
const names = queryParams.name.split(',');
```

Reply:
Makes sense, change made :)